_episodes/13-scheduler.md

```diff
@@ -126,6 +137,103 @@ hostname
+to tell Bash exactly where to look. To run a script that we wrote ourselves, we
+need to tell Bash where it is explicitly, so it doesn't go searching. We could
+do this one of two ways: either with the absolute path 
+`{{ site.workshop_host_homedir }}/yourUserName/example-job.sh` (equivalently,
+`~/example-job.sh`), or with the relative path `./example-job.sh`.
+
+```
+$ ./example-job.sh
+```
+{: .language-bash}
+
+```
+bash: ./example-job.sh: Permission denied
+```
+{: .error}
+
+There's one last thing we need to do. Before a file can be run, it needs
+"permission" to execute. Let's look at our file's permissions with `ls -l`:
+
+```
+$ ls -l *.sh
+```
+{: .language-bash}
+
+```
+...
+-rw-rw-r-- 1 yourUsername yourGroup  40 Jan 16 19:41 example-job.sh
+...
+```
+{: .output}
+
+That's a huge amount of output: a full listing of everything in the directory.
+Let's see if we can understand **what each field of a given row represents**,
+working left to right.
+
+1. **Permissions:** to the far left, there are 10 single-character columns with
+   some or all of the following symbols:
+   * `d` indicates whether something is a directory
+   * `r` indicates permission to **r**ead the file or directory
+   * `w` indicates permission to **w**rite to the file or directory
+   * `x` indicates permission to e**x**ecute the file or directory
+   * `-` is used where permission has not been granted.
+
+   There are three fields of `rwx` permissions following the spot for `d`:
+  
+   * The first set of `rwx` are the permissions that the `u`ser who owns the
+     file has (in this case the owner is `yourUsername`).
+   * The second set of `rwx`s are permissions that other members of the owner's
+     `g`roup share (in this case, the group is named `yourGroup`); shorthand is `g`.
+   * The third set of `rwx`s are permissions that all `o`ther users can do with
+     a file.
+   * Although files are typically created with read permissions for
+     everyone, typically the permissions on your home directory prevent others
+     from being able to access the file in the first place.
+2. **References:** This counts the number of references ([hard links](
+   https://en.wikipedia.org/wiki/Hard_link)) to the item (file, folder,
+   symbolic link or "shortcut").
+3. **Owner:** This is the username of the user who owns the file. Their
+   permissions are indicated in the first permissions field.
+4. **Group:** This is the user group of the user who owns the file. Members of
+   this user group have permissions indicated in the second permissions field.
+5. **Size of item:** This is the number of bytes in a file, or the number of
+   [filesystem blocks](https://en.wikipedia.org/wiki/Block_(data_storage))
+   occupied by the contents of a folder. (We can use the `-h` option here to
+   get a human-readable file size in megabytes, gigabytes, etc.)
+6. **Time last modified:** This is the last time the file was modified.
+7. **Filename:** This is the filename.
```

Comment:
I like this explanation! I think there are also some good diagrams around to represent `ls-l` output format - perhaps one could be reused here?

Reply:
I've seen a few enumerations of columns, like this one, but can't recall a good diagram or image doing it justice. If you come across one, please let me know, and if it's Creative Commons licensed, I'll include or hotlink it!